sycl/include/CL/sycl/types.hpp

```diff
@@ -579,7 +579,8 @@ template <typename Type, int NumElements> class vec {
   static constexpr size_t get_count() { return NumElements; }
   static constexpr size_t get_size() { return sizeof(m_Data); }
 
-  template <typename convertT, rounding_mode roundingMode>
+  template <typename convertT,
+            rounding_mode roundingMode = rounding_mode::automatic>
```

Comment:
You make this method different from the specification. Why?
```
template <typename convertT, rounding_mode roundingMode>
vec<convertT, numElements> convert() const;
```

Reply:
Hmm.. i think this is missing in the spec.. I'll ask for them to update it..
Table 4.98 for rounding mode automatic says:
Default rounding mode for the SYCL vec class element type element type, rtz (round toward zero) for integer types and rte(roundtonearesteven)forﬂoating-point types.

Reply:
If I do not do this change, I have to call convert method with two template arguments:
convert<cl_int, rouding_mode::default> , even though I want to use the default rounding mode.

Reply:
> Hmm.. i think this is missing in the spec.. I'll ask them to update it..

Yes, please follow our procedure, make an proposal into the specification.

> If I do not do this change, I have to call convert method with two template arguments:
> convert<cl_int, rouding_mode::default> , even though I want to use the default rounding mode.

Unfortunately, I also do it in my patch.